lab/src/Microsoft.ServiceModel.Syndication/tests/SampleTest.cs

```diff
@@ -10,6 +12,76 @@ public static class SampleTest
         [Fact]
         public static void PassingTest()
         {
+           
+        }
+        
+        [Fact]
+        public static void crear()
+        {
+            SyndicationFeed sf = new SyndicationFeed("First feed on .net core ever!!", "This is the first feed on .net core ever!", new Uri("https://microsoft.com"));
+
+            //xml writter
+            XmlWriter xmlw = XmlWriter.Create("FirstFeedEver.xml");
+            Rss20FeedFormatter rssf = new Rss20FeedFormatter(sf);
+            rssf.WriteTo(xmlw);
+            xmlw.Close();
+            Assert.True(true);
```

Comment:
>Assert.True(true) [](start = 12, length = 17)

`Assert.True(true)` will never fail. Please assert on something we can verify. Please find @jcondex or @mconnew for more suggestions.